src/commands/deploy/nextjs.ts

```diff
@@ -52,22 +52,19 @@ export async function nextJsDeploy(options: GenezioDeployOptions) {
 
     checkProjectLimitations();
 
-    // Deploy NextJs serverless functions
-    const deploymentResult = await deployFunctions(genezioConfig, options.stage);
-
-    // Deploy NextJs static assets to S3
-    const domainName = await deployStaticAssets(genezioConfig, options.stage);
-
-    // Set environment variables for the Next.js project
-    await setupEnvironmentVariables(deploymentResult, domainName);
+    const [deploymentResult, domainName] = await Promise.all([
+        // Deploy NextJs serverless functions
+        deployFunctions(genezioConfig, options.stage),
+        // Deploy NextJs static assets to S3
+        deployStaticAssets(genezioConfig, options.stage),
+    ]);
 
-    // Deploy CDN that serves the Next.js app
-    const cdnUrl = await deployCDN(
-        deploymentResult.functions,
-        domainName,
-        genezioConfig,
-        options.stage,
-    );
+    const [, cdnUrl] = await Promise.all([
```

Comment:
I don't necessarily like this syntactic sugar, but maybe it's just me.

I think it would be a bit more readable to rewrite:
```js
const deploymentArtefacts = await Promise.all([...])

const cdnUrl = deploymentArtefacts[1]
```

Reply:
It's a personal preference. I for example like the syntactic sugar because it's a one liner. I don't think I will change this.